### PR TITLE
Other configurable options

### PR DESCRIPTION
**Pull request by adeptofvoltron**
car

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-  - nightly
-
+- '7.0'
 before_script:
-  - composer self-update
-  - composer install --prefer-dist --no-interaction
-
+- composer self-update
+- echo "{\"http-basic\":{\"repo.magento.com\":{\"username\":\"${MAGENTO_USERNAME}\",\"password\":\"${MAGENTO_PASSWORD}\"}}}" > auth.json
+- composer install --prefer-dist
```
Comment (pradzikowski) [inline .travis.yml:8]: forget no-interaction ?

File: Command/Phinx/ConfigBuilder.php [hunk 2]
```diff
@@ -90,7 +89,7 @@ protected function getDbConfig()
                 'name' => $config['dbname'],
                 'user' => $config['username'],
                 'pass' => $config['password'],
-                'port' => 3306,
+                'port' => empty($config['port']) ? 3306 : $config['port'],
```
Comment (pradzikowski) [inline Command/Phinx/ConfigBuilder.php]: IMO default config should come from xml :/

File: Command/Phinx/ConfigBuilder.php [hunk 3]
```diff
@@ -90,7 +89,7 @@ protected function getDbConfig()
...
                 'charset' => 'utf8'
```
Comment (pradzikowski) [inline Command/Phinx/ConfigBuilder.php:93]: Still not configurable charset

File: README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ Creating tool for migrations in magento, with `rollback` option.
 ### Download
 You can add these module into your project by adding lines below into your composer.json `require` section:
 ```
-        "pcf/magento-migration" : "dev-master",
```
Comment (pradzikowski) [inline README.md:12]: Imo module is still under development

**Review by pradzikowski — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (adeptofvoltron) [reply README.md:12]: but working version is already downloadable.

**Comment by pradzikowski** [comment]
AND WHERE FOR GOD SAKE IS GIF WITH CAT !?

File: README.md (hunk 4 above)
Comment (pradzikowski) [reply README.md:12]: downloadable didnt mean it is stable and functional :)

**Review by pradzikowski — APPROVED** [review]

**Comment by pradzikowski** [comment]
@adeptofvoltron promised me to fix all missed things. They are not urgent, tests passed so i can merge them.

**Pull request merged**